lib/mosaic-utils/src/main/java/org/eclipse/mosaic/lib/util/gson/DataFieldAdapter.java

```diff
@@ -72,7 +73,23 @@ public void write(JsonWriter out, Long param) throws IOException {
         if (param != null && param == Long.MAX_VALUE) {
             out.value(UNLIMITED);
         } else {
-            out.value(ObjectUtils.defaultIfNull(param, 0L));
+            long value = ObjectUtils.defaultIfNull(param, 0L);
+            String unitPrefix = "";
+
+            if (value % DATA.TERABIT == 0) {
```

Comment:
should we also test for `== 0` right at the beginning? otherwise we would export `0` to `"0 Tb"` instead of `"0 b"`. It's still correct, though, but counterintuitive. The same for TimeFieldAdapter.